### DISCUSSION TITLE
CC: Tweaked Compat Pls

### DISCUSSION THREAD

**Discussion by AmooWasHere** [ChoiceTheorem/ChoiceTheorem-s-overhauled-village#41] [Ideas]
no rush not really integral to mod but would be nice

**Comment by ChoiceTheorem** [comment]
Will consider after I finished my hiatus!

**Comment by ChoiceTheorem** [comment]
Done! 
[Curseforge](https://curseforge.com/minecraft/texture-packs/ctov-advanced-peripheral-compat)
[Modrinth](https://modrinth.com/datapack/ctov-advanced-peripheral-compat)

**Comment by AmooWasHere** [comment]
thanks!
